### ISSUE TITLE
keyboard navigation does not work in debug console

### ISSUE THREAD

**Issue by akosyakov** [theia-ide/theia#5493]

**Comment by akosyakov** [comment]
I don't have time to have a look, feel free to grab.

**Comment by tsmaeder** [comment]
@akosyakov I can navigate the rows with the arrow keys no problem. Can you clarify what's wrong, please?

**Comment by akosyakov** [comment]
To reproduce:
- put a breakpoint in the last test in uri.spec.ts
- use `Run Mocha Test` to run this file
- scroll down when the breakpoint hits
- select the last row and try navigate up with the keyboard
- the viewport should be always updated to reveal the selected row, but it is not

**Comment by tsmaeder** [comment]
An even simpler scenario:
1. Add lines to the debug console by just typing into the command prompt to make sure you have some invisible items
2. select some visible line
3. Arrow-up until the selection should disappear.
